A/Atomsk/build_tarballs.jl

```diff
@@ -7,25 +7,35 @@ version = v"0.11.2"
 
 # Collection of sources required to complete build
 sources = [
-    GitSource("https://github.com/pierrehirel/atomsk.git", "3333858281e0ebd6279825b83ab871cf4d050a8d")
+    GitSource("https://github.com/pierrehirel/atomsk.git", "84f60a20c5b814fec03bffe1bccc3daaed0fc65d"),
```

Comment:
Changed the source without changing the version number

Reply:
This is fixed in the latest commit.